docs/docs/queries/structure.md

```diff
@@ -1,44 +1,207 @@
 # Structure of a Query
 
-The dataview query language is a simple, structured, custom query language for quickly creating views on your data. It
+Dataview provides [multiple ways](dql-js-inline.md) to create queries and the way you need to write a query differs for each.
+
+This page provides information on how to write a **Dataview Query Language** (**DQL**) query. If you're interested in how to write Inline Queries, refer to the [inline section on DQL, JS and Inlines](dql-js-inline.md#inline-dql). You'll find more information about **Javascript Queries** on the [Javascript Reference](../api/intro.md).
+
+**DQL** is a SQL like query language for creating different views or calculations on your data. It
 supports:
 
-- Fetching pages associated with tags, folders, links, and so on.
-- Filtering pages / data by simple operations on fields, like comparison, existence checks, and so on.
-- Sorting results based on fields.
+- Choosing a **output format** of your output (the [Query Type](./query-types.md))
+- Fetch pages **from a certain [source](../reference/sources.md)**, i.e. a tag, folder or link
+- **Filtering pages/data** by simple operations on fields, like comparison, existence checks, and so on
+- **Transforming fields** for displaying, i.e. with calculations or splitting up multi-value fields
+- **Sorting** results based on fields
+- **Grouping** results based on fields
+- **Limiting** your result count
 
-The query language supports the following view types, described below:
+Let's have a look at how  we can put DQL to use.
 
-1. **TABLE**: The traditional view type; one row per data point, with several columns of field data.
-2. **LIST**: A list of pages which match the query. You can output a single associated value for each page.
-3. **TASK**: A list of tasks whose pages match the given query.
-4. **CALENDAR**: A calendar view displaying each hit via a dot on its referred date
+## General Format of a DQL Query
 
-Read more about the available types [here](./query-types.md)
+Every query follows the same structure and consists of
 
-## General Format
+- exactly one **Query Type** with, depending on your Query type, zero, one or many [fields](../annotation/add-metadata.md) for displaying
+- zero or one **FROM** data command with one to many [sources](../reference/sources.md)
+- zero to many other **data commands** with one to many [expressions](../reference/expressions.md) and/or other infos depending on the data command 
 
-The general format for queries is:
+Abstractly speaking, a query conforms the following pattern:
```

Comment:
Can be simplified: a query typically conforms to the following pattern

Reply:
Perhaps:

At a high level, a query conforms to the following syntax/format/pattern